compute-budget-instruction/src/compute_budget_instruction_details.rs

```diff
@@ -31,6 +50,7 @@ pub struct ComputeBudgetInstructionDetails {
     // Additional builtin program counters
     num_builtin_instructions: u16,
```

Comment:
can we add a comment on this field, or change the name, so that it's clear that this count is only for builtins that are not possibly migrated?

The math in `calculate_default_compute_unit_limit` makes a lot more sense if that is clearer 😄

Reply:
`s/num_builtin_instructions/num_non_migratable_builtin_instructions/g`